src/core/sys/freebsd/execinfo.d

```diff
@@ -43,7 +43,13 @@ else
             import ldc.llvmasm;
             __asm("str x29, $0", "=*m", &p);
         }
+        else version (PPC64) // LDC
+        {
+           import ldc.llvmasm;
+           __asm("stdu %r1,-144(%r1)", "=*m", &p);
```

Comment:
Can you explain this or link to a reference implementation? It looks like 'store stack pointer (%r1) into %r1-144' and as such an extremely brittle way to set `p`; `stdu %r1, $0` would probably make more sense in that case.

Reply:
I didn't create this code, I pulled from an earlier upstream update that was not committed.  I updated the code based on your comments.

Reply:
Okay, do you have a link? - And so you haven't tested this at all?

Reply:
See dialog in #3112 concerning the code mentioned above.  You may also want to confer with Redstar.

Yes, I've tested.  You can see comments in the PR #3112

Please also refer to https://github.com/dlang/druntime/pull/2683